docs/source/release/v5.1.0/diffraction.rst

```diff
@@ -5,42 +5,53 @@ Diffraction Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
 Powder Diffraction
 ------------------
 New features
 ^^^^^^^^^^^^
-- New ``D7`` instrument definition for ILL
-- New :ref:`SaveGSSCW <algm-SaveGSSCW>` is implemented for constant wavelength diffractometers.
+- New ``D7`` instrument definition for ILL.
+- New :ref:`SaveGSSCW <algm-SaveGSSCW>` algorithm is implemented for constant wavelength diffractometers.
 
 Improvements
 ^^^^^^^^^^^^
-- Polaris.create_total_scattering_pdf output workspaces now have the run number in the names.
-- Polaris.create_total_scattering_pdf no longer takes `output_binning` as a parameter, instead binning of the output pdf can be controlled with `delta_r`.
-- Polaris.create_total_scattering_pdf can rebin the Q space workspace before calculating the PDF by being given an input `delta_q`.
-- SampleDetails.set_materials now differentiates between sample density and crystal density for converting between pdf types.
-- :ref:`LoadWAND <algm-LoadWAND>` now adds `duration` log to the workspace
-- Polaris.create_total_scattering_pdf now no longer calculates the PDF with the Lorch filter enabled by default and must be enabled.
-- :ref:`PDFFourierTransform <algm-PDFFourierTransform>` has been updated to allow for the algorithm to be run on a PDF to obtain a spectrum density function.
-- :ref:`CalculatePlaczekSelfScattering <algm-CalculatePlaczekSelfScattering>` now accepts the crystalographic density of the sample to correct for the powder density.
-- Square beam profile of 5mm x 5mm added to the PEARL_Definition_new_lowangle instrument definition file
-- running `Polaris.create_total_scattering_pdf` with `debug=true` will preserve the `self_scattering_correction` workspace.
+- The output workspace names from `Polaris.create_total_scattering_pdf` now include run numbers.
```

Comment:
Where there are 4-5 lines relating to the same algorithm (or method in this case) it would be nice to show the changes as sub-bullets below a main bullet ie
- Various improvements have been made to `Polaris.create_total_scattering_pdf`:

  - output workspaces names now include run numbers
  - control of the output binning of … etc

Am putting these in as comments rather than requested changes because not sure how late I am. Not worth holding up the release for!